app/templates/components/modals/change-image-modal.hbs

```diff
@@ -0,0 +1,50 @@
+<i class="black close icon"></i>
+<div class="header">
+  {{t 'Update image'}}
+</div>
+<br>
+<br>
+<div class="ui grid">
+  <div class="row">
+    <div class="sixteen wide column">
+      {{widgets/forms/image-upload
+      
+      needsCropper=true
+      label=(t 'Update Image')
+      id='user_image'
+      icon='photo'
+      hint=(t 'Select Image')
+      maxSizeInKb=10000
+      helpText=(t 'For Cover Photos : We recommend using at least a 1300x550px (2:1 ratio) image.
+                                                      For Avatar Photos : We recommend using at least a 360x360px (1:1 ratio) image.
+                                                      The images should be no larger than 10MB.') class='center aligned'}}
+      <br>
```

Comment:
Please remove all br tags. In most cases if you need to use this, it probably means you're doing some wrong with layout. 

Just see the basic form markup at https://semantic-ui.com/collections/form.html 

Please follow a proper form markup. 

```hbs
<form>
   <div class="field">
       <label>Label</label>
       <input>
    </div>
   <div class="field">
       <label>Label</label>
       <input>
    </div>
   <div class="field">
       <label>Label</label>
       <input>
    </div>

</form>
```